test/eks/lock.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: lock
+spec:
+  replicas: 0
+  selector:
+    matchLabels:
+      app: lock
+  template:
+    metadata:
+      labels:
+        app: lock
+    spec:
+      containers:
+      - name: lock
+        image: alpine
```

Comment:
specify repository and version for this image

Reply:
It doesn't matter, this deployment won't ever create any pods.